samza-kafka/src/main/scala/org/apache/samza/util/KafkaUtil.scala

```diff
@@ -133,54 +133,6 @@ class KafkaUtil(val retryBackoff: ExponentialSleepStrategy = new ExponentialSlee
   }
 
   /**
-   * Common code to validate partition count in a topic
-   *
```

Comment:
Why not delete the createTopic method too?

Reply:
hm. You are right. It can be deleted too. Will do.